client/src/components/carousel.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+
+/*
+Ways to implement this module:
+1.  Conditional rendering. Load multiple sets of restaurant cards and
+    render them as you click back/forward
+2.  handleClickBack/Next will load the next 3 cards or previous 3 
+3.
+*/
+
+class Carousel extends React.Component {
+  // constructor() {
```

Comment:
constructor and super should have argument props buddy. please add them, if you are thinking about using a state to hold your pictures.